### PR TITLE
Layout update to search results page and table 

### PR DESCRIPTION
**Pull request by brianfleming**
## Description
- Update the overall page layout to better match the designs used in the new shared feed page and settings areas
- Introduce "pinned" checkbox column so users can always select and item and also see what items are selected regardless of horizontal scroll
- Introduce a new unread color for stronger visual indication of read status
- Update row hover color to design system hover color

## Type of change

- [x] Performance improvement and/or refactoring (non-breaking change that keeps existing functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Automated test (add or update automated tests)

## How has this been tested?
 manual testing

### AFTER - page layout
Screenshot 2023-09-01 at 4 16 16 PM

### AFTER - table scrolling vert and horizontal
Screenshot 2023-09-01 at 4 16 32 PM

### AFTER - pinned checkbox column on horizontal scroll
Screenshot 2023-09-01 at 4 16 45 PM

### AFTER - unread rows get a left border color
Screenshot 2023-09-01 at 4 17 00 PM

## Checklist

- [x] I have performed a self-review of my own code
- [x] I've made sure my branch is runnable and given good testing steps in the PR description
- [ ] I have commented my code in hard-to-understand areas, if any
- [ ] I have made needed changes to the README
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If I implemented any new components, they are self-contained, their `propTypes` are declared and they use React Hooks and, if data-fetching is required, they use Relay Modern with fragment containers
- [x] I have removed the /* eslint-disable @calm/react-intl/missing-attribute */ from any files I've worked on and added a `description` prop to all `<FormattedMessage />` components that were missing it.
- [x] To the best of my knowledge, any new styles are applied according to the design system
- [ ] If I added a new external dependency, I included a rationale for doing so and an estimate of the change in bundle size (e.g., checked in https://bundlephobia.com/)

### CODE REVIEW

File: src/app/components/media/ReportDesigner/ReportDesignerComponent.js [hunk 1]
```diff
@@ -227,7 +227,7 @@ const ReportDesignerComponent = (props) => {
         onStateChange={(action, state) => { handleSave(action, state); }}
         prefixUrl={prefixUrl}
       />
-      <Box display="flex" width="1">
+      <Box display="flex" width="1" style={{ borderLeft: 'solid 2px var(--grayBorderMain)' }}>
```
Comment (brianfleming) [inline src/app/components/media/ReportDesigner/ReportDesignerComponent.js:230]: This was added temporarily, to keep the item page looking the same until we get around to those layout changes

File: src/app/components/media/ReportDesigner/ReportDesignerTopBar.js [hunk 2]
```diff
@@ -32,6 +32,7 @@ const useStyles = makeStyles(theme => ({
   },
   reportHeader: {
     backgroundColor: 'var(--brandBackground)',
+    borderLeft: 'solid 2px var(--grayBorderMain)',
```
Comment (brianfleming) [inline src/app/components/media/ReportDesigner/ReportDesignerTopBar.js:35]: This was added temporarily, to keep the item page looking the same until we get around to those layout changes

File: src/app/components/Header.js [hunk 3]
```diff
@@ -14,6 +14,7 @@ import {
 
 const HeaderBar = styled.div`
   background-color: var(--brandBackground);
+  border-left: solid 2px var(--grayBorderMain);
```
Comment (brianfleming) [inline src/app/components/Header.js:17]: This was added temporarily, to keep the item page looking the same until we get around to those layout changes

File: src/app/components/Home.module.css [hunk 4]
```diff
@@ -14,14 +14,18 @@
...
 
 .mainContentWrapper {
-  background-color: var(--otherWhite);
+  background-color: var(--grayBackground);
   display: flex;
   flex-direction: column;
   height: 100%;
   width: 100%;
 
+  &:global(.route__media),
```
Comment (brianfleming) [inline src/app/components/Home.module.css:23]: This was added temporarily, to keep the item page looking the same until we get around to those layout changes

File: src/app/components/media/MediaComponent.js [hunk 5]
```diff
@@ -23,6 +23,7 @@ import { getSuperAdminMask } from '../../helpers';
 import { units, Column } from '../../styles/js/shared';
 
 const StyledThreeColumnLayout = styled.div`
+  border-left: solid 2px var(--grayBorderMain);
```
Comment (brianfleming) [inline src/app/components/media/MediaComponent.js:26]: This was added temporarily, to keep the item page looking the same until we get around to those layout changes

**Review by caiosba — APPROVED** [review]
Looks great! I noticed a bug but it's also happening on `develop`:

Captura de tela de 2023-09-01 20-44-12

So I'm going to file it separately.

**Comment by caiosba** [comment]
> Looks great! I noticed a bug but it's also happening on `develop`:
> 
> Captura de tela de 2023-09-01 20-44-12
> 
> So I'm going to file it separately.

I pushed a fix here: https://github.com/meedan/check-web/pull/1606

**Pull request merged**